SWADroid/src/main/res/xml/preferences.xml

```diff
@@ -25,6 +25,22 @@
             android:summary="@string/prefSyncTimeSummary"
             android:title="@string/prefSyncTimeTitle" />
     </PreferenceCategory>
+
+    <PreferenceCategory android:title="@string/prefSyncLocationTitle">
+        <CheckBoxPreference
+            android:defaultValue="false"
+            android:key="@string/prefShareLocationKey"
+            android:title="@string/prefShareLocation" />
+
+        <ListPreference
+            android:defaultValue="5"
+            android:entries="@array/prefSyncLocationTimeEntries"
+            android:entryValues="@array/prefSyncLocationTimeValues"
+            android:key="@string/prefSyncLocationTimeKey"
+            android:summary="@string/prefSyncLocationTimeSummary"
+            android:title="@string/prefSyncLocationTimeTitle" />
```

Comment:
I would use the same title as for the synchronization of the notifications for homogeneity.

```suggestion
            android:title="@string/prefSyncTimeTitle" />
```